docs/installing-northstar/northstar-installers/vtol-guide.md

```diff
@@ -56,56 +56,57 @@ To expand the smaller version of the tab list, press the button with 3 bars as s
 
 ![3 Bars](../../images/vtol-settings-expand.png)
 
-**NOTE: This guide will not be including the "Home" button as a tab when describing this list. When it says the first tab, it means the first tab underneath the home button**
+__NOTE: This guide will not be including the _"Home"_ button as a tab when describing this list. When it says the first tab, it means the first tab underneath the home button__
```

Comment:
```suggestion
**NOTE: This guide will not be including the _"Home"_ button as a tab when describing this list. When it says the first tab, it means the first tab underneath the home button**
```

Again I assume you wanted that line to still stay bold? Double underscore is not properly defined in vanilla markdown IIRC. Discord uses it for underlining a sentence but not sure if that also applies to GitBook...